app/models/concerns/policies/early_career_payments/eligible.rb

```diff
@@ -9,8 +9,27 @@ def award_amount
         super || BigDecimal(calculate_award_amount || 0)
       end
 
+      def journey
+        @journey ||= Journeys.for_policy(policy)
+      end
+
+      def journey_session
+        @journey_session ||= journey::Session.find(claim.journeys_session_id)
```

Comment:
I don't think we can use this association here, as the claim isn't associated with the journey session until the claim is submitted and once this migration has finished we wont create a claim until the end of the journey when the claim submission form is submitted.
Also it's still up for debate if we even want to associate the claim and the journey session, or if we just throw the journey session away once the claim has been created